### PR TITLE
feat: synchronized timer between players in lobby

### PR DESCRIPTION
**Pull request by daniellonda**
Players now select a timer duration and "ready up" to start the timer. both players recieve the same duration and start at the same time.

### CODE REVIEW

File: Assets/_Scripts/LocaleSelector.cs
Comment (vhuynh100) [inline Assets/_Scripts/LocaleSelector.cs:1]: can you revert this change since I already removed this in main

Comment (daniellonda) [reply Assets/_Scripts/LocaleSelector.cs:1]: > can you revert this change since I already removed this in main

no
